src/routes/safe/components/Apps/confirmTransactions.js

```diff
@@ -38,9 +48,31 @@ const confirmTransactions = (
   closeModal: () => void,
   onConfirm: () => void,
 ) => {
+  const isTxError = txs.some((t) => {
```

Comment:
the function you pass as an argument could be extracted to a separate one like `validateTransaction`

Reply:
done!